postgres2.yml

```diff
@@ -1,106 +1,78 @@
-ttl: &ttl 30
-loop_wait: &loop_wait 10
-scope: &scope batman
+scope: batman
+#namespace: /service/
+name: postgresql2
+
 restapi:
   listen: 127.0.0.1:8010
   connect_address: 127.0.0.1:8010
-  auth: 'username:password'
 #  certfile: /etc/ssl/certs/ssl-cert-snakeoil.pem
 #  keyfile: /etc/ssl/private/ssl-cert-snakeoil.key
+  authentication:
```

Comment:
I think acceptance tests should not even rely on these "sample" config files but generate them from scratch. With [environment](https://github.com/zalando/patroni/blob/master/docs/ENVIRONMENT.rst) configuration these configs would be really small and in most of the cases even not necessary... Basically config file is needed only when you are bootstrapping the new cluster or you want to configure some "tags"